### DISCUSSION TITLE
i deserve mod

### DISCUSSION THREAD

**Discussion by nekoify** [aeolus-1/triangleGame.github.io#71] [General]
i am pro coder which means i need mod so I don't have to pr every time I make small update

**Comment by nekoify** [comment]
upvote this

**Comment by Quantaviousdingleberry420** [comment]
LX8369 does deserve mod because of his contribution to multiplayer 🙏

**Reply by nekoify** [reply]
i agree

**Comment by serifonium** [comment]
yes
